xonsh/environ.py

```diff
@@ -1675,6 +1667,14 @@ def DEFAULT_VARS():
             "Logging to a file is not enabled by default.",
         ),
     }
+    if hasattr(locale, "LC_MESSAGES"):
+        dv["LC_MESSAGES"] = Var(
```

Comment:
@melund - is this correct?

Reply:
It is not related to this PR. Instead it belongs to the discussion here https://github.com/xonsh/xonsh/pull/3658#issuecomment-670959034

Reply:
So it should probably be in its own PR